07-civic_overview.Rmd

```diff
@@ -17,6 +17,23 @@ The core concept of CIViC is an Evidence item linked to a Source (PubMed, ASCO o
 ```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The core element of CIViC is the evidence item (or EID) which includes an unstructured, human-readable evidence statement and multiple structured elements briefly describing a key piece of evidence from the biomedical literature related to the clinical interpretation of cancer variants. Evidence is linked to a molecular profile (variant) for a specific gene, disease, and optional phenotype. Evidence is further categorized into 6 types and 5 levels as shown."}
 ottrpal::include_slide("https://docs.google.com/presentation/d/1zwvAVLWpN2mKrJLgo5ZcAuSKQrHZ8zunIYHLe-w6bBo/edit#slide=id.g22509046821_0_14")
 ```
+## CIViC Somatic Assertions
+
+```{r, fig.align='center', out.width="100%", echo = FALSE, fig.alt= "The core element of CIViC is the evidence item (or EID) which includes an unstructured, human-readable evidence statement and multiple structured elements briefly describing a key piece of evidence from the biomedical literature related to the clinical interpretation of cancer variants. Evidence is linked to a molecular profile (variant) for a specific gene, disease, and optional phenotype. Evidence is further categorized into 6 types and 5 levels as shown."}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1zwvAVLWpN2mKrJLgo5ZcAuSKQrHZ8zunIYHLe-w6bBo/edit#slide=id.g22509046821_0_14")
+```
+
+## CIViC Somatic Assertions
+
+CIViC Assertions summarize a collection of Evidence Items for a given variant and disease. Furthermore, Assertions associate a guideline based evaluation to the variant within the given disease context. Evidence for a variant can come from multiple types of published reports. For some established variants, evidence will have been published describing preclinical research, case studies, clinical trails, and regulatory approvals. In this case, a higher tier will be applied. For instance, a Predictive Assertion describing the sensitivity of EGFR L858R to afatinib in non small cell lung cancer will receive AMP/ASCO/CAP Tier I Level A, since this treatment is FDA approved for the given variant and disease. Assertions of variant oncogenicity will evaluate evidence from multiple sources, including CIViC Evidence Items, as well as data reported from other resources such as the gnomAD database of population frequencies. The summation of this collection of evidence will result in an oncogenicity evaluation based on the ClinGen/CGC/VICC guidelines.   
```

Comment:
```suggestion
CIViC Assertions summarize a collection of Evidence Items for a given variant and disease. Furthermore, Assertions associate a guideline-based evaluation to the variant within the given disease context. Evidence for a variant can come from multiple types of published reports. For some established variants, evidence will have been published describing preclinical research, case studies, clinical trails, and regulatory approvals. In this case, a higher tier will be applied. For instance, a Predictive Assertion describing the sensitivity of EGFR L858R to afatinib in non small cell lung cancer will receive AMP/ASCO/CAP Tier I Level A, since this treatment is FDA approved for the given variant and disease. Assertions of variant oncogenicity will evaluate evidence from multiple sources, including CIViC Evidence Items, as well as data reported from other resources such as the gnomAD database of population frequencies. The summation of this collection of evidence will result in an oncogenicity evaluation based on the ClinGen/CGC/VICC guidelines.   
```